helper/convert/type.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package convert
+
+import (
+	"github.com/hashicorp/go-cty/cty"
+	"github.com/hashicorp/terraform-plugin-go/tftypes"
+)
+
+// An private version of this function to convert cty.Type to tftypes.Type exists in
+// internal/plugin/convert/schema.go but this is a simplified and public version
+// that maps the types one to one without any error handling, under the assumption
+// the incoming data will always be valid
```

Comment:
Does this function need to be duplicated and public? If it's only being consumed by `ResourceData` then perhaps we could just re-use that internal one and remove this?

Or if we need this exported version we could call the internal one and ignore/panic on the error?